### DISCUSSION TITLE
(Monthly View) Show Only The Current Month's Dates

### DISCUSSION THREAD

**Discussion by valiantgenomics** [davish/obsidian-full-calendar#408] [Feature Requests]
I was wondering if it would be possible to toggle/remove all the dates surrounding the current month, which also show up in the current month's view. Essentially the past dates roll over into the new month, and the next month's dates roll back, again into the current month.

Example below:

![image](https://user-images.githubusercontent.com/111633318/205480309-52b52c7a-fb6b-48ee-ae58-ffe8fe0e5a0f.png)

I think there's a lot of power in isolating each month and just focusing on that month itself, without seeing past and future dates. For me personally, it creates a ton of confusion when I'm seeing past dates at the very top of the monthly view and future dates at the very bottom - almost as if these events are pending for this month, when they are not.

So would it be possible to make those boxes, shown in the screenshot above, just blank black boxes without dates&events, just to hide them from the current month and isolate the current month?

To isolate each month and keep it self-contained, without any rolling forwards and backwards of dates.

Thanks a ton!

**Comment by davish** [comment]
This is possible with the fullcalendar option described here: https://fullcalendar.io/docs/showNonCurrentDates

**Comment by valiantgenomics** [comment]
@davish Yes, that seems very straightforward! Thanks a lot for pointing it out! It's precisely what I was looking for!

I'm just wondering how exactly can I set this variable to "false"?

**Comment by valiantgenomics** [comment]
Just bumping this up.
